### DISCUSSION TITLE
Cannot run multiple servers on the same system

### DISCUSSION THREAD

**Discussion by Odin303** [apohl79/audiogridder#973] [Q&A] [answered]
Hello,

I tried to find some information but couldn't 

I'm actually using AG 1.2 beta 10 on the same system with my daw (Studio one & win10). Everything runs OK, except few plugins not recognized.

I would like to run multiple servers on the same system :

1. How do you create multiple servers? If I understand well, I have to specify different IDs (+name to help to Differentiate) in server settings, then save. Right? 
2. Then, inside the plugin, I have to add the servers I previously created, right? But what must I specify in the field? I tried "ServerName:Server ID" or "ip address:Server ID" but it doesn't connect! 

Any help please? 

Then, is there any user manual/guide/tutorial available? Couldn't find one. 

Cheers

**Comment by apohl79** [comment, marked as answer]
You have to run the server manually from the command line passing the "-id" parameter, if you want to use different servers on the same machine. For example, open a powershell prompt and run:

```
& 'C:\Program Files\AudioGridderServer\AudioGridderServer.exe' -id 1
& 'C:\Program Files\AudioGridderServer\AudioGridderServer.exe' -id 2
& 'C:\Program Files\AudioGridderServer\AudioGridderServer.exe' -id 3
```

This will run 3 servers with ids 1-3. Each server will broadcast its existence to clients (ag plugins in your DAW) on your network, so you will see them in the server menu and you usually do not have to add them manually.

**Comment by Odin303** [comment]
Thank you very much for your reply. Now, it works flawlessly!
